.travis.yml

```diff
@@ -8,16 +8,24 @@ matrix:
       os: osx
       osx_image: xcode8.2
       before_install:
+        - rvm list
+        - rvm install .ruby-version 
+        - bundle install
         - cd Utility || exit
```

Comment:
Isn't `Utility` always be there? If it's the case, I don't see the need to be protective with `|| exit`.